src/open_inwoner/scss/components/_index.scss

```diff
@@ -26,6 +26,7 @@
 @import './Form/ButtonRadio.scss';
 @import './Form/ButtonRadio.scss';
 @import './Form/Checkbox.scss';
+@import 'Form/ChoiceList';
```

Comment:
is this equivalent to `@import './Form/ChoiceList.scss;`? I'm assuming yes, because it works :p

Reply:
Oops - I need to correct this - not sure if I am jeopardizing Webpack here :-)